cluster/mesos/docker/mesos-slave/bin/wrapdocker

```diff
@@ -0,0 +1,166 @@
+#!/bin/bash
+
+# Copyright 2013-2014 Jérôme Petazzoni, Johan Haleby, lalyos, James Harris,
```

Comment:
We might need to figure out in parallel what to do about this file if this copyright is odd. What's its original provenance? And it's definitely originally [Apache v2](http://www.apache.org/licenses/LICENSE-2.0) but you're remixing it here? I think non Kubernetes Authors contributions typically end up in third_party - everything in the main repo is by someone that's signed the CLA, but I need to poke at it.